hw4/moduleSort/moduleSort.c

```diff
@@ -0,0 +1,88 @@
+﻿#pragma warning (disable : 6031)
```

Comment:
Компилятор пишет: `moduleSort.c(48,18): warning C4996: 'fopen': This function or variable may be unsafe. Consider using fopen_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.`
Поэтому попробовать так:
```suggestion
﻿#pragma warning (disable : 4996)
```
может быть хорошей идеей. По крайней мере, у меня получилось :)